com.unity.ml-agents/CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
```

Comment:
Can we link to https://github.com/Unity-Technologies/ml-agents/blob/master/CONTRIBUTING.md (or replace this with that, and link to here)?

Reply:
done